tests/vmi_configuration_test.go

```diff
@@ -48,6 +48,10 @@ import (
 	"kubevirt.io/kubevirt/tests"
 )
 
+const (
+	dmidecodePackageUrl = "http://download-ib01.fedoraproject.org/pub/fedora/linux/releases/29/Everything/x86_64/os/Packages/d/dmidecode-3.2-1.fc29.x86_64.rpm"
```

Comment:
This is a pattern which we can't have in our test suite. We can't rely during the test run on a package which can any time disappear, or where we have to reach out to the internet.

Can you please have a look at https://github.com/kubevirt/kubevirt/tree/master/images/cdi-http-import-server? We use it for instance to server the guest-agent for our fedora image while testing.

You basically have to download the rpm in the WORKSPACE file, and then place it in the cdi-http-import-server. Let me know if you have questions.